src/mapping/config/MappingConfiguration.cpp

```diff
@@ -257,7 +257,7 @@ MappingConfiguration::ConfiguredMapping MappingConfiguration::createMapping(
   } else if (direction == VALUE_READ) {
     configuredMapping.direction = READ;
   } else {
-    PRECICE_ERROR("Unknown direction type \"" << direction << "\"!");
+    PRECICE_ASSERT(false, "Unknown mapping direction type \"" << direction << "\". Please check the documentation for available options.");
```

Comment:
I am using a (doomed to fail) `PRECICE_ASSERT` here, as this is already checked by the XML parser. I kept the suggestion vague to avoid duplication (we may add more options in the future).